### PR TITLE
[REF] web: convert views/view_tests to hoot

### PR DESCRIPTION
**Pull request by Polymorphe57**
Task ID: 3705027

### CODE REVIEW

**Review by aab-odoo — APPROVED** [review]

File: addons/web/static/tests/views/view.test.js [hunk 1]
```diff
@@ -0,0 +1,1371 @@
...
+        search: /* xml */ `<search/>`,
+        "search,1": /* xml */ `
+            <search>
+                <filter name="filter" domain="[(1, '=', 1)]"/>
+                <filter name="group_by" context="{ 'group_by': 'display_name' }"/>
+            </search>
+        `,
+    };
+
+    filters = [
```
[the remark this answers is not in the document]
Comment (Polymorphe57) [reply addons/web/static/tests/views/view.test.js:74]: maybe we could change that of course. In that case we should just change that
        if (kwargs.options.load_filters && "search" in views) {
            views["search"].filters = this.filters || [];
        }
Note that I am not a big fan of that (same thing for _views). I would prefer to have a model ir.filters

File: addons/web/static/tests/views/view.test.js [hunk 2]
```diff
@@ -0,0 +1,1371 @@
...
+    makeMockEnv,
+    mockService,
+    models,
+    mountWithCleanup,
+    onRpc,
+    patchWithCleanup,
+} from "@web/../tests/web_test_helpers";
+import { click, queryOne } from "@odoo/hoot-dom";
+import { Deferred, animationFrame, runAllTimers } from "@odoo/hoot-mock";
+import { pick } from "@web/core/utils/objects";
```
[the remark this answers is not in the document]
Comment (Polymorphe57) [reply addons/web/static/tests/views/view.test.js:20]: can you recall me how you sort imports? I think you mean a partition helpers/others but what else?

[the remark this answers is not in the document]
Comment (aab-odoo) [reply addons/web/static/tests/views/view.test.js:20]: at least separate hoot imports (global) from suite imports (coming from module set, specific to this suite)

**Review by aab-odoo — APPROVED** [review]
robodoo r+

File: addons/web/static/tests/views/view.test.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Polymorphe57) [reply addons/web/static/tests/views/view.test.js]: I have use the prefix _

**Pull request closed without merging**